src/libcore/ptr.rs

```diff
@@ -39,6 +39,9 @@ pub use intrinsics::copy;
 #[stable(feature = "rust1", since = "1.0.0")]
 pub use intrinsics::write_bytes;
 
+#[unstable(feature = "drop_in_place", reason = "just exposed, needs approval")]
```

Comment:
In addition to what @huonw said about this needing `issue=` now this'll also need to be applied to the definition in the intrinsics module itself to work (we don't look at reexports themselves)